lib/index.js

```diff
@@ -26,19 +31,19 @@ function preprocessor(css, options) {
 
   options = options || {};
 
-  var browserConfig = options.browsers || [
-    'Explorer >= 9',
-    'last 2 Chrome versions',
-    'last 2 Firefox versions',
-    'last 2 Safari versions',
-    'last 2 iOS versions',
-    'Android 4'
-  ];
-
-  css = rework(css, options).use(suit(options)).toString(options);
-
-  // vendor prefixes
-  css = autoprefixer(browserConfig).process(css).css;
+  css = postcss([
+    atImport({
+      root: options.root,
```

Comment:
I was relying on `postcss-import` to deal with falsey values, which I think it does.